--- old
+++ new
@@ -56,0 +56,8 @@
+
+# general full stack developer español
+Querido gestor de contratación.
+Soy un desarrollador de software con conocimientos en desarrollo de aplicaciones multiplataforma.
+Durante mi tiempo con Flykube fui responsable de la integración de nuevos diseños y el desarrollo de código web permitiendo a la empresa verse profesional y confiable. También, cuando se me requería, ayudaba al equipo de backend a arreglar bugs o implementar nuevas funciones ahorrando tiempo y dinero a la compañia.
+Soy un apasionado por la programación algo que me hace buscar nuevas maneras de escribir código.
+Me enfoco siempre en escribir un código limpio fácil de leer, escribir y mantener, buscando a la vez la estructura de datos, paradigma, patrón o algoritmo que encaje con el código en particular.
+