--- old
+++ new
@@ -12,1 +12,3 @@
-Soy José Zaquinaula, programador de aplicaciones multiplataforma y 
+Soy José Zaquinaula, programador de aplicaciones multiplataforma y desarrollo el código de un videojuego. Ayudo a los equipos de videojuegos que producen productos de realidad pero necesitan  personal capacitado para hacerlo. Al unirme a su equipo aporto capacidad, creatividad, innovación y soluciones para desarrollar el código necesario para un videojuego, siguiendo estandares y colaborando con el equipo.
+Lo que hago es utilizar mis conocimientos que he adquirido como programador a lo largo de los casi 7 años como desarrollador de aplicaciones multiplataforma habiendo especializado recientemente en el área de los videojuegos.
+Si te parece interesante y te gustaría contactar conmigo o saber más de mi te ofrezco mi LinkedIn y página web