--- old
+++ new
@@ -6,0 +6,1 @@
+## Version 1 
@@ -10,0 +11,21 @@
+## Version 2
+señores y señoritas, paginas web.
+Odio al gobierno, es la verdad, cada vez que entro a una de sus paginas me dan dolor de cabeza.
+Cl@ve,La meva Salut, Mobilitat, Renfe, Policia Nacional 
+Han hecho un tramite a traves de la web? Tienes que tener un diploma para poder manejar eso.
+Y peor aun los negocios particulares, te sale mas rapido ir a la tienda que esperar a que cargue la pagina.
+yo no soy ningun experto para criticar...no espera si que lo soy, muchas cosas se arreglan con un simple linea que diga margin 2 puntos center.
+Ahora, paginas como https://noclip.website/, la conocen? pues es una pagina donde puedes cargar mapas enteros de viejuegos en segundos!
+o https://dustinbrett.com/ que te carga un sistema operativo completo, un ordenador entero en un solo link, hasta tiene doom 
+en realidad no odio al gobierno, quiero decir, trabaje para ellos por 3 años. 
+En realidad queria hacerles ver que vale la pena tener personas que sean apasionados en lo que hacen, y no solo que hayana aprendido cosas. 
+Yo me considero alguien encantado por la programacion, y una prueba ingfalible es que sigo a midudev y fireship.
+Soy conciente que cada año salen nuevas cosas y que hay que mantenerse flexible, y en constante aprendizaje, hace 1 año no se pensaba en poder crear una pagina en segundos, ahora con Cursor puedes hacer una en segundos.
+
+Pienso que una pagina web esta bien hecha si se siguen las buenas practicas de cada lenguaje, cuando hay un equipo organizado, se planifica bien, y que siempre va a ver cosas que puedas mejorar, un tester lo sabe muy bien.
+
+Quiero decir, he hecho 5 paginas web enteras yo solo, he creado 6 app para android y estoy en procesos de crear 1 app multiplataforma, he conectado aplicaciones web y moviles. APIs, bases de datos, librerias, frameworks.
+Eso si, estoy especializado en ReactJs y JS en general. 
+Yo me esmero en crear la mejor pagina web posible basado en lo que quieras mostrar, un blog, un servicio, ecommerce, applicaciones, apis.
+
+