--- old
+++ new
@@ -5,1 +5,1 @@
-Con esto, se crean aplicaciones estables, de fácil manteniendo y de gran calidad, de manera que puedan lograr sus objetivos y mantenerlos en el tiempo. Lo que hago es utilizar mis conocimientos que he adquirido a lo largo de los casi 7 años programando como desarrollador front-end, backend, y freelancer.
+Con esto, se crean aplicaciones estables, de fácil manteniendo y de gran calidad, de manera que puedan lograr sus objetivos y mantenerlos en el tiempo. Lo que hago es utilizar mis conocimientos que he adquirido a lo largo de los casi 7 años programando como desarrollador front-end y backend tanto de manera particular como profesional.