--- old
+++ new
@@ -10,0 +10,6 @@
+2. **Nota simple**: resumen del estado de la vivienda. Ej: 
+   - bonita y todo pero...es vivienda  proteccion oficial (tiene condiciones para vivir o alquilarla)
+   - ta guapo pero...está de usufructo, tons no puedes vivir ahí hasta q el otro se retire
+   - promoción casa barata + una demanda judicial 
+   - maravilla de piso, solo que en papel me equivoque del número catastral, ahora soy dueño de un baño en un edificio de Macdonald
+   - 