--- old
+++ new
@@ -12,1 +12,1 @@
-- Valor intrinseco: es el valor verdadero de una accion. No es el precio si no el valor obtenido con el analisis fundamental.
+- Valor intrinseco: es el valor verdadero de una accion. No es el precio si no el valor obtenido con el analisis fundamental. Es el valor maximo que tendrá la accion en su vida, calculando sus añadidos cada año hasta el dia de su muerte.
@@ -118,0 +118,37 @@
+
+## Empresas con beneficios predecibles
+### Estable
+mantiene el valor de una accion en unos limites estables. ej. de 9 a 11 USD desde su inicio
+$benefico medio = \frac{año1+año2+año3}{totalAños}$
+### Ciclica
+el valor de las acciones dibujan un ciclo de precio. ej. un ciclo de cada 4 años donde oscila de 1, 5, 10, 15
+$benefico medio = \frac{año1+año2+año3}{añosCiclo}$
+### Crecimiento
+el valor va creciendo de manera constante.  ej. crece cada año +3
+$crecimiento = \sqrt[1/totalAños]{ultimoAño/primerAño}$
+$benefico siguiente año = añoAnterior * crecimiento$
+$año n = primerAño * \sqrt[n]{1 + crecimiento}$
+
+## Buenos Valores de los Ratios
+- Margen Bruto(beneficio bruto): >20%
+- PER ? = 2xCRECIMIENTO + 8.5
+- CR: > 1.5
+- Ratio de solvencia: > 1.5
+- ROE: >15% ( tomar en cuenta patrimonio neto corregido)
+### otros valores
+#### Balance General
+- activo: 
+    - efectivo: demasiado es malo, si es que es un hecho puntual. Malo: mucho efectivo + deuda o acciones
+    - inventario: crecimiento proporcional a ventas
+- pasivo:
+    - deuda a L.P : < 4XbeneficioNeto
+    - prima de emision: - no debe aumentar mucho cada año.es el exeso que se paga por el sobreprecio de una accion.  Empresas que aumentan las acciones tienen mucho de esto. ej. tesla
+- patrimonio: 
+    - reservas acumuladas: + acumulacion de efectivo. Se decrece es malo.
+    - acciones propias: - compra de sus propias acciones. aumenta el valor de las acciones, accion que hacen las buenas empresas.
+    - patrimonio neto: si es negativo, se debe colocar las acciones propias en + y volver a sumar. eso crea el patrimonio neto corregido.
+    
+#### estado de resultados
+- gastos operativos en gestion, administracion: debe manternerse lo mas constante posible en el tiempo. Son costes de transporte, alquiler, servicios basicos.  
+- beneficio neto: >10%, y con un crecimiento constante
+![](valoresBuenosRatios.jpg)