--- old
+++ new
@@ -166,2 +166,6 @@
-6. Ratios de liquidez-solvencia
-7. 
+6. Ratios de liquidez-solvencia. % deuda y con el fcf cuanto tiempo pagará deuda.
+7. % crec. Patrimonio, Retenciones, # acciones
+8. Estado resultados: % crec. Ingresos, mgn. Bruto, % de investigación y mrg. Operativo.,%  de ingresos financieros con ventas y % crec. .mrgn. neto
+9. Estados de flujo: % capex (activ. De inversion), recompra inv. y emisión de ventas.
+10. Graficas de 10años: EPS, Free c.f.,
+11. Free cash flow/acción, ROE y comp. Con su nicho