--- old
+++ new
@@ -19,0 +19,21 @@
+
+--- 
+
+Dear Hiring Manager,
+
+I am excited to apply for the programmer internship position at Rare. As someone deeply passionate about the gaming industry, I recently completed a one-year specialization course in video game development, which greatly expanded my knowledge and inspired me to pursue this path professionally.
+
+As a programmer, I explored various fields such as web development, app creation, systems programming, and cybersecurity during my education. However, it was clear from the start that video games were the perfect blend of creativity and technical challenge for me.
+
+My passion for creating experiences began in childhood when I made animations and stories about my friends which were wildly entertaining(or at least that’s what their polite laughter suggested). I started with Pivot Animator, then Adobe Flash, where I also experimented with ActionScript to create simple games. Later, during my bachelor’s degree, I ventured further into programming, making primitive games with the Linux terminal and a tile-based graphics game using Java’s Swing library. That project in particular sparked a lasting excitement for game development.
+
+These days, I spend my free time creating games in Unreal Engine and Unity, whether for jams, hackathons, or personal projects. Occasionally, I also use Blender to create animations, rekindling my love for storytelling in visual form.
+
+Rare has always been a company I deeply admire. Its legacy of creating memorable experiences like Donkey Kong—a game that gave me countless joyful moments with my cousin—and the visually stunning Everwild inspires me. Joining your team would be an incredible honor.
+
+While I have built a strong technical foundation, I am eager to learn the collaborative processes, tools, and methodologies used in professional game development. I would greatly value the opportunity to work with experienced mentors, contribute meaningfully to your projects, and support Rare’s mission to create unforgettable games.
+
+Thank you for considering my application. I look forward to the opportunity to discuss how I can grow with Rare and contribute to its creative vision.
+
+Sincerely,
+José Alejandro Zaquinaula Ramon