--- old
+++ new
@@ -13,1 +13,2 @@
-Software Engineer with experience in cross-platform application development seeking to get involved in . I am eager to find a job as gameplay engineer where i can put all my skills to use while also learning from industry professionals to grow into a excellent gameplay engineer.
+Software Engineer with experience in cross-platform application development seeking to get involved in game development.
+I am eager to explore the diferent areas of the game development I am eager to find a job as gameplay engineer where i can put all my skills to use while also learning from industry professionals to grow into a excellent gameplay engineer.