--- old
+++ new
@@ -6,2 +6,2 @@
-* Joan*: es por eso que me inventé varias formas para evitar comer su comida. La primera que hago es contarle algo. Cuando se da cuenta me toca recurrir a hacer un trato. Y ya si no funciona tampoco chantaje emocional y listo.
-* Deisy*: ese se cree que no me sé sus trucos. ( B-roll comiendo y Joan haciendo sus trucos y la Deisy con cara de póker ) Si no se inventa un historia de 1h, se pone a hacerme trueques de esto a cambio de esto y si no se pone a chillar rogandome que no quiere mas.
+*Joan*: es por eso que me inventé varias formas para evitar comer su comida. La primera que hago es contarle algo. Cuando se da cuenta me toca recurrir a hacer un trato. Y ya si no funciona tampoco chantaje emocional y listo.
+*Deisy*: ese se cree que no me sé sus trucos. ( B-roll comiendo y Joan haciendo sus trucos y la Deisy con cara de póker ) Si no se inventa un historia de 1h, se pone a hacerme trueques de esto a cambio de esto y si no se pone a chillar rogandome que no quiere mas.