--- old
+++ new
@@ -29,0 +29,2 @@
+## my voice version
+F<sup>4</sup> C<sup>4</sup> A#<sup>4</sup> F<sup>4</sup>
@@ -30,0 +32,1 @@
+Dm<sup>2</sup> C<sup>2</sup> A#<sup>2</sup> F<sup>2</sup>
@@ -31,0 +34,19 @@
+Gm<sup>4</sup> C<sup>2</sup> C/G<sup>2</sup> X2
+
+CORO:
+
+F<sup>4</sup> C<sup>4</sup> A#<sup>4</sup> F<sup>4</sup> X2
+
+C<sup>4</sup>
+
+F<sup>4</sup> Dm<sup>4</sup> Am<sup>4</sup> A#<sup>4</sup>
+
+C<sup>4</sup> F<sup>4</sup> Dm<sup>4</sup> Am<sup>4</sup>
+
+Dm<sup>4</sup> Am<sup>4</sup> A#<sup>4</sup> C<sup>4</sup>
+
+F<sup>4</sup>
+
+
+
+