--- old
+++ new
@@ -215,1 +215,4 @@
-Ahora que lo tengo todo para mí 
+Ahora que lo tengo todo, dinero, trabajo, tiempo libre, amigos. No puedo parar este ardor, y ya no se de que depende.
+Tengo la esperanza que sea por que estoy inhibido, que estar aquí de extranjero rodeado de personas que quieren mucho a su bandera me intimida y me hace tener vergüenza de estar ahí. Pero al mismo tiempo se que me tratan bien, que no les molesta mi presencia, y que los chistes pesados sobre mi origen y acento son eso, chistes, algo que aquí es normal burlarse de eso y algo que John Cleese aplaude, bromear de uno mismo debería ser saludable, pero mi corazón no lo siente así, siente que se están metiendo con él con su dignidad y que debe alejarse para que ellos puedan estar tranquilos...y me vuelvo gracias a eso, irónicamente, justo una persona que no quieren a su lado, no ya por su origen sino por su personalidad cerrada y mímica. 
+Y que hago yo? Como le hago entender la realidad? O como hago pa tener el valor de vencer esa fuerza invisible que me inutiliza, la que me impide ser como soy yo y me limita seguir movimientos repetitivos y de una persona que no soy yo.
+En mis pensamientos de vez en cuando sale la idea de esperar a mi héroe/heroína que me salve de esto, que eleve mi ser y le haga ver la realidad, y que me ayude a coger fuerzas para vencer aquella otra que me tumba.