--- old
+++ new
@@ -212,0 +212,4 @@
+
+## Dolores del Alma otra vez
+Que es lo que sucede?. Exactamente qué es lo que me disparaba en el pecho y me destroza todo por dentro?. Puede ser el monstruo del no tener a nadie con quién hablar de mis cosas, o el sentirme que nadie me desea, o puede ser el imposible de relacionarme con la gente de manera cómoda. O tal vez estos monstruos han sido expulsados del embrión de otro más grande que no conozco?.
+Ahora que lo tengo todo para mí 