--- old
+++ new
@@ -224,1 +224,5 @@
-En fin, que no tengo claro cuál fue mi base 
+En fin, que no tengo claro cuál fue mi base
+
+## Mi corazon no sabe que quiere
+Me entusiasma el hecho de tener una pareja, no voy a mentir, pero al mismo tiempo me angustia.
+Quiero que alguien me haga emocionar de despertarme en la mañana y coger el móvil para ver unos buenos días. Quiero que alguien le haga cumplidos a mi ser, "eres una persona maravillosa", "eres lo más bonito que ma ha pasado", "solo pienso en ti". Quiero que al estar enfermo o tener mi alma debilitada llamar a alguien que enrralidad se preocupe por mí y que me alivie el sufrimiento. Quiero que mi vida se mezcle con la vida de otra persona a tal punto que los hilos de esas historias se crucen tanto que no se puedan separar jamás. Quiero liberar este bulto de ganas de amar, proteger, consolar. Quiero sentir una confianza infinita con alguien a tal punto que le confiere todos mis más pesados miedos sin siquiera preocuparme en lo que pensara de mi.