--- old
+++ new
@@ -222,1 +222,3 @@
-O puedo ir más atras
+O llendo un poco más atrás, cuando en último grado de escuela un niño decidió pegarme por maldad genuina, por crear conflicto, y yo me opuse a regresar el golpe, siendo un canon para mí personalidad futura.
+O tal vez la interacción con el gato y las historias que me contaba sobre su vida en el colegio hizo que creamos juntos dos roles para cada uno, siendo uno diferente al otro, siendo el el sociable y yo el menos sociable.
+En fin, que no tengo claro cuál fue mi base 