--- old
+++ new
@@ -3,1 +3,1 @@
-Juaquin: Es que ese el problema, que a mi me gustan todas. Cada que paso por una calle me enamoró de una mujer, y no solo de mi edad sabes. Es que para mí todas son lindas y no tengo ni que elegir, cualquiera me puede llegar a enamorar.
+Juaquin: Es que ese el problema, que a mi me gustan todas. Cada que paso por una calle me enamoró de una lady, y no solo de mi edad sabes. Es que para mí todas son lindas y no tengo ni que elegir, cualquiera me puede llegar a enamorar.
@@ -5,1 +5,1 @@
-Juaquin: "tu por ejemplo", lo casi grito señalando a Vero. "Tu me gustaste la primera vez que te ví. Esa pinche cara toda guapa que tienes, parecías un angel de ojos penetrantes color del cielo, y cuando hablaste con ese acento canario buaa, me volaste la mente."
+Juaquin: "tu por ejemplo", lo casi grito señalando a Vero. "Tu me gustaste la primera vez que te ví. Esa pinche cara toda guapa que tienes, parecías un angel con esos ojos penetrantes color del cielo, que tu no lo notas porque son tus ojos pero hipnotizas con ellos.Y cuando hablaste con ese acento canario buaa, me volaste la mente."
@@ -9,1 +9,1 @@
-Juaquin: "si claro. Volviendo a lo que decía, me gustabas y me ilusionaba mucho hablar contigo por chat, pero ya el tiempo ayuda a ver lo mortal que son ustedes, por que no, no  es como me lo hacen creer al principio, no son angeles, son pinches demonios con un disfraz carisimo".
+Juaquin: "si claro. Volviendo a lo que decía, me gustabas y me ilusionaba mucho hablar contigo por chat la verdad, pero por suerte el tiempo ayuda a ver lo mortal que son ustedes, por que no, no  es como me lo hacen creer al principio, no son angeles, son pinches demonios con un disfraz carisimo".