--- old
+++ new
@@ -27,1 +27,1 @@
-- cuando fui a cagar y no me pillaron donde cague, y le cogieron el fusil al sevilla
+- cuando fui a cagar y no me pillaron donde cague, y le cogieron el fusil al ortega
@@ -40,1 +40,1 @@
-    - poncho de emergencia y el sevilla con hipotermia
+    - poncho de emergencia y el ortega con hipotermia
@@ -47,1 +47,1 @@
-    - pensar mucho en los q me quieres y como me tratan ( finguen tratar) los mandos mientras monto con iago los bloques de hielo
+    - pensar mucho en los q me quieren y como me tratan ( finguen tratar) los mandos aqui mientras monto con iago los bloques de hielo