--- old
+++ new
@@ -115,2 +115,7 @@
-    - 
-    - 
+    - día del helicóptero, formamos en la mañana y fuimos hacia dentro, casco y pa dentro del helicóptero, salte y flote por la ropa, di mi pulgar de todo correcto, llegue al primero y Asensi se dio cuenta q me iba, vino el bote y nos llevó al puerto, Camarena nos enseño como flotar con la ropa, me ahogaba pero ahí seguí, Romualdo trataba de enseñarme, después con el srg primero sabe rcomo usar la mochila para flotar. Falle 
+    - regresar a la casa, hablarnos como amigos y decirme Séneca por decir q leo filosofía, llegan todos y empezamos a cargar cosas, nos querían meter la idea de q nos íbamos 
+    - subimos al bus pero era un engaño al dar vuelta Vázquez con una sirena nos grito alair del bus y sacamos las mochilas de debajo y fomar, directo al agua con la mochila y hacer ejercicios por 1hora, salir y volver otra vez, fari y ruso pagando por los amigos. Pasar reparando por el camión e ir a los Conguitos , descanzar, quitarse la ropa y dormir dentro del saco...obligados, nos juro que no nos Ivana despertar
+    - 1hra después nos despiertan con alarmas y muchos ruidos y 5min pa vestirse con la misma ropa mojada y guardar todo en la mochila, nos mojan y a patear todo el perímetro, volvér a casa y obligarnos a dormir sin ropa, y así 2veces más 
+    - amaneció y nos llevan en camión al agua, a volver a nadar con mochila, no me movía no podía patear para moverme, salir y coger Bernardos, havermaniobres de combate desde ahí hasta la casa, fingir q acabo, lanzar una Granada y gritarnos ir por debajo del camión reptando, pasa rla mini pista de aplicación, llegar a edificios de la base y colgarnos de un árbol, media hora de esfuerzos, finalmente "quien quiera la boina aquí está cojanla", hicimos piña y juntos fuimos a por ella, CElma llora y los instructores nos dan la enhorabuena
+    - pizza en la casa, dormir alado de Asensi 
+    - uniforme de deporte y Pal bus 