--- old
+++ new
@@ -64,1 +64,1 @@
-    - pista de estres
+    - raplin en un pueblo, pasar en una cuerda encima de un río, pasarlo después caminando mojados todos, cambiarse y llevar la bolsa de eripa mojada desde el río hasta el camión. Llegar y sacarnos a gritos del bus y ordenarnos ponernos la ropa mojada ( hacia mucho frío ), esperar media hora a q se pongan todos la ropa, empezar a hacer rondas de bebox y que siga el primero Miranda ( esto no siquiera empieza ) y llevarnos a la pista de estrés 
@@ -93,0 +93,2 @@
+     - en el bus comer el Chiricico bueno que pasaban por los asientos ( Iago hablando de su casa de barro )
+     