--- old
+++ new
@@ -27,1 +27,1 @@
-- cuando fui a cagar y no me pillaron donde cague, y le cogieron el fusil al sevilla
+- cuando fui a cagar y no me pillaron donde cague, y le cogieron el fusil al ortega
@@ -40,1 +40,1 @@
-    - poncho de emergencia y el sevilla con hipotermia
+    - poncho de emergencia y el ortega con hipotermia
@@ -47,1 +47,1 @@
-    - pensar mucho en los q me quieres y como me tratan ( finguen tratar) los mandos mientras monto con iago los bloques de hielo
+    - pensar mucho en los q me quieren y como me tratan ( finguen tratar) los mandos aqui mientras monto con iago los bloques de hielo
@@ -59,2 +59,2 @@
-    - pintarse en el espejo ya armado con el chaleco ese y el uniforme
-    - comer como imbeciles e ir con sevilla a esa panaderia de las palmeras de chocolate blanco y los churros, devorar el a400
+    - pintarse en el espejo ya armado con el peco y el uniforme
+    - comer como imbeciles e ir con Ortega a esa panaderia de las palmeras de chocolate blanco y los churros, devorar el a400
@@ -62,0 +62,1 @@
+    - 1era semana de defensa personal