--- old
+++ new
@@ -66,0 +66,1 @@
+    - ir al sushi a comer como cerdos
@@ -67,0 +68,1 @@
+    - 
@@ -94,0 +96,5 @@
+- semana pos-kukalooh:
+    - volver a dormir en hab, formar sin peco pa ir a defensa personal, limpiar en armeria
+    - el primero gallego se. Volvió loco y los últimos días nos mojaba con la cantimplora
+    - un día no limpie bien el arma, me llevo a bebox, después se animaron y hicieron revista de mochilas donde mojaron a todos, hicimos varias vueltas a la plaza cantando Asensi.
+    - 