--- old
+++ new
@@ -67,0 +67,14 @@
+ - kukalloh:
+     - llegar a una zona plana con colinas pequeñas
+     - formacion circular y después quitarnos las mochilas y dejarnos una vieja con 3 cosas(poncho, Gore-Tex, saco)
+     - mojar al almelloneh
+     - hipotermia al ortega en la paridera
+     - perder la antena de la radio 
+     - en una subida de montaña tirarse al vacío cuando venía un coche de lo alto
+     - dormir en una carpa natural hecha anteriormente, el peor frío no dormir nada y muy apretados, binomio era Moya
+     - hacer lo de escalar y aprender carpas con paracaídas
+     - montar guardias, mejorar tiendas y esperar hasta la noche, si te quedabas quieto te daba hipotermia ( como el Sanse)
+     - pateada, pasamos alado de un molino, un coche nos vio y nos tumbamos en un hueco
+     - dormir en otra paridera todos amontonados, dormir con fari y ruseins
+     - despertar y hacer lo del sospechosos infiltrado
+     - 