--- old
+++ new
@@ -94,0 +94,1 @@
+     - recoger cosas de la casa donde dormían los instructores, había un advertising de show clown o algo así en donde pasaban los días 
@@ -100,0 +101,14 @@
+    - última pateada con Vázquez en Épila, ( Sanse, Moya, Ortega y yo ), pasamos por molinos, vimos las luces del ala 31 a lo lejos, una malla nos estorbo el paso, subimos al camión y nos tapamos con la manta, esperamos al resto, al llegar nos mojaron.
+    - por ir mojados a defensa personal , nos mojaron dos veces y nos quedamos así todo el día ( Ortega, Iago y yo )
+    - semana de god , montar otra vez la zona de aislamiento
+    - practicar casos de ataque ( gas, fuego, ataque aéreo, ide o mochila desconocida a lo lejos, soldado herido, carta sospechosa )
+    - día de inglés, Iago se durmió en una guardia y mojaron a todos ...me salve
+    - cerveza en armeria con todos después de hacernos mojado el primero 
+    - noche más terrorífica, miedo a no saber q nos pueden hacer pero si saber. Vamos. Sufrir
+- Murcia:
+    - 8 horas de viaje, el 25 lloraba de miedo atrás de mi asiento por no saber q nos iban a hacer 
+    - gritarnos para salir, en 1min nos mojaron llevándonos al mar, esperamos instrucciones y nos llevaron a la casa esa.
+    - descargar cosas y fui el primero en mojarse por no responder bien las preguntas 
+    - 1er día dos mojadas de dia, 1de noche, cambio de puestos cada 2 horas, el mejor era el q te mandaba lejos de esa casa.
+    - tiritar de frío, la ropa se pegaba al cuerpo y un movimiento que hacías te daba escalofríos, fari y yo de bin
+    - guardia de puerta 1era mojada por los q no hicieron bien el relevo, por no encontrar la ide Sanse, todos al agua
@@ -101,2 +116,1 @@
-     
-
+    - 