--- old
+++ new
@@ -77,1 +77,4 @@
-     - pateada, pasamos alado de un molino, un coche nos vio y nos tumbamos en un hueco
+     - pateada, ver figuras en la noche que se movían, en una cuesta dura llegamos a dormir, montamos un poncho rápidamente y dormimos ( sin dormir ) menos apretados pero con frío 
+     - aprender a hacer trampas de caza, almacenar agua con una bolsa de basura, nrbq
+     - montar puesto y guardia encima de una colina hasta que se haga la noche, todos encima de todos con mantas americanas encima, en mi guardia se acercaron a llamarnos pa salir a la pateada
+     - en una cima Ortega se equivocó de camino, vimos a la otra patrulla cuando la teníamos a 10m ( muy oscuro) y vilariño enojado con la extremera, pasamos alado de un molino, un coche nos vio y nos tumbamos en un hueco, en cada parada montar la radio
@@ -79,2 +82,11 @@
-     - despertar y hacer lo del sospechosos infiltrado
-     - 
+     - despertar y hacer lo del sospechoso infiltrado
+     - subirá un camión y salir de él con gritos y montar formación, pateada hasta que nos empiezan a disparar
+     - llegada a otra paridera con algunos heridos dejados atrás, monte guardia con la minimi en la calle con López de binomio con su bazuka
+     - recoger a los heridos con camillas mientras se los cubría 
+     - atender heridos en la paridera con los conocimientos de clase
+     - matar y comer conejos con tomillo
+     - incio de la pista de estrés kukalon, dándonos 5min para guardar todo en la mochila cuando justo antes nos trataron como amigos.
+     - repté por un río como castigo, después empezó la pista, troncos, pasada por un caudal,debajo del camión, pasada por una trinchera subida y bajada de una camilla..3veces
+     - al terminar nos pusieron en una pared a recostar nos llenos de barro, y encima de los guantes de mierda nos colocaron comida mezclada de nuestros equipos ( con hambre lo primero entro, después ya no)
+     - cambiar ropa en el fuego, todos desnudos, López pasando con sus tetillas por el fuego, Iago bailando bachata
+     - recoger cosas al camión y subir al bus