--- old
+++ new
@@ -5,1 +5,1 @@
-Hoy ví una pareja sentada en frente mío en el tren, la chica recostó su cabeza en el asiento evinclino la cabeza para ver a su pareja, el sin notar su mirada cuidaba al bebé. La mirada tan fuera de este mundo de la chica me enterneció, lo veía como si se tratase de un ángel.
+Hoy ví una pareja sentada en frente mío en el tren, la chica recostó su cabeza en el asiento e inclinó la cabeza para ver a su novio, él,sin notar la mirada,cuidaba al bebé. La mirada tan fuera de este mundo de la chica me enterneció, lo veía como si se tratase de un ángel.
@@ -7,1 +7,1 @@
-Para mi la belleza femenina es bella y que se me conceda el derecho de observarla por horas me parece el regalo más preciado que se me pueda dar, porque no es un regalo para el cuerpo ni para la mente, sino para el alma, al que tan poco le damos.
+La belleza de una mujer me llena de vitalidad y que se me conceda el derecho de observarla por horas me parece el regalo más preciado que se me pueda dar, porque no es un regalo para el cuerpo ni para la mente, sino para el alma, al que tan poco le damos.