--- old
+++ new
@@ -5,1 +5,1 @@
-I'm eager to get involved in the game industry world, I'm recently finished a one year course about videogames. There are web, app, systems, network, cyber security, videogames and others I may not know, and I feel videogames is the one for me. Since I was a child I was doing animations, telling stories about my friends, then showing them and  see them laugh. Then I tried to make videogames, with Adobe flash, but was difficult in that time for me. Now in my free time, after entering in the programming world, I've made animations with blender, and a few games in Java, Js and  Unreal Engine
+I'm eager to get involved in the game industry world, I'm recently finished a one year course about videogames. For a programmer there are webs, apps, systems, network, cyber security, videogames and perhaps others I may not know, and I feel videogames is the one for me. Since I was a child I was doing animations, telling stories about my friends, then showing them and  see them laugh. Then I tried to make videogames, with Adobe flash, but was difficult in that time for me. Now in my free time, after entering in the programming world, I've made animations with blender, and a few games in Java, Js and  Unreal Engine
@@ -9,1 +9,1 @@
-I have good technical skills but I don't know how to make a very good videogame or how it is to be part of it. I would like to have a mentor that guides me in my first steps in the industry and learn about all the programming roles
+I have good technical skills but I don't know how to make a very good videogame or how it is to be part of it. I would like to have a mentor that guides me in my first steps in the industry and learn about all the programming roles.
@@ -13,0 +13,2 @@
+
+