--- old
+++ new
@@ -5,1 +5,1 @@
-I'm eager to get involved in the game industry world, I'm recently finished a one year course about videogames. For a programmer there are webs, apps, systems, network, cyber security, videogames and perhaps others I may not know, and I feel videogames is the one for me. Since I was a child I was doing animations, telling stories about my friends, then showing them and  see them laugh. Then I tried to make videogames, with Adobe flash, but was difficult in that time for me. Now in my free time, after entering in the programming world, I've made animations with blender, and a few games in Java, Js and  Unreal Engine
+I'm eager to get involved in the game industry world, I'm recently finished a one year course about videogames.
@@ -7,0 +7,2 @@
+For a programmer there are diferents webs, apps, systems, network, cyber security, videogames and perhaps others I may not know, and I feel videogames is the one for me. Since I was a child I was doing animations, telling stories about my friends, then showing them and  see them laugh. Then I tried to make videogames, with Adobe flash, but was difficult in that time for me. Now in my free time, once entering in the programming world, I make animations with blender, and a few games in Java, Js and  Unreal Engine
+
@@ -11,0 +13,1 @@
+Thanks for give me your time for reading this.
@@ -14,1 +17,0 @@
-