--- old
+++ new
@@ -8,1 +8,1 @@
-Then I tried to make videogames, using the same Adobe flash and its ActionScript programming language, but it was difficult in that time for me. So once I entered in the programming world, I made very primitive games with the Linux terminal, and then try the 
+Then I tried to make videogames, using the same Adobe flash and its ActionScript programming language, but it was difficult in that time for me. So once I entered in the programming world, I made very primitive games with the Linux terminal, and then try the  GUI java library call Swing