--- old
+++ new
@@ -8,2 +8,1 @@
-Then I tried to make videogames, using the same Adobe flash and its ActionScript programming language, but it was difficult in that time for me. So once I entered in the programming world, I made very primitive games with the Linux terminal, and then try the  GUI java library call Swing with which I made few games.
-Now in my free time , I make animations with blender, and a few games in Java, Js and  Unreal Engine
+Then I tried to make videogames, using the same Adobe flash and its ActionScript programming language, but it was difficult in that time for me and left it there. So once I entered in the programming world, I made very primitive games with the Linux terminal, and then try the  GUI java library call Swing with which I made a project to create a tile-base graphics game. I get excited with that project.
@@ -11,1 +10,1 @@
-Epic games is a big company that have made amazing videogames like Fortnite or Gears of Wars franchise and for me it's an honor to work in Epic Games. 
+So now in my free time , once in a while I make animations with blender, but using more time making  games in  Unreal Engine mainly, whether it is for a jam, hackaton or for fun.
@@ -13,1 +12,1 @@
-I have good technical skills but I don't know how to make a very good videogame or how it is to be part of it. I would like to have a mentor that guides me in my first steps in the industry and learn about all the programming roles.
+Epic games is a big company that I really appreciate, it have made amazing videogames like Fortnite or Gears of Wars franchise and for me it's an honor to work in Epic Games.
@@ -15,0 +14,2 @@
+I have good technical skills but I don't know how to make a very good videogame or how it is to be part of it or what kind of process or  methods to use. I would like to have a mentor that guides me in my first steps in the industry and learn about all the programming roles, and of course to make all the effort to make good products and 
+